src/Styles/AuthorizationStyles/authorization-styles.scss

```diff
@@ -0,0 +1,163 @@
+$font-family: 'Roboto';
+$font-size: 14px;
+
+$color-primary: #aba194;
+
+.user
+{
+    width: 90%;
+    max-width: 340px;
+    margin: 10vh auto;
+}
+
+.user__header
+{
+    transform: translate3d(0, 500px, 0);
+    animation: arrive 500ms ease-in-out .7s forwards;
+    text-align: center;
+
+    opacity: 0;
+}
+
+.user__title
+{
+    font-size: $font-size;
+    font-weight: 500;
+
+    margin-bottom: -10px;
+
+    color: white;
+}
+
+.form
+{
+    overflow: hidden;
+
+    margin-top: 40px;
+
+    transform: translate3d(0, 500px, 0);
+    animation: arrive 500ms ease-in-out .9s forwards;
+
+    opacity: 0;
+    border-radius: 6px;
+}
+
+.form__input
```

Comment:
use scss features:
you can create nested styles:

.form {
    &__input {
    }
}